test/e2e/tsdx-build-default.test.ts

```diff
@@ -41,6 +41,13 @@ describe('tsdx build :: zero-config defaults', () => {
     expect(output.code).toBe(0);
   });
 
+  it('should add shebang', async () => {
+    const output = execWithCache('node ../dist/index.js build');
+    expect(output.code).toBe(0);
+    const matched = grep(/#!\/usr\/bin\/env/, ['dist/build-default.*.js']);
```

Comment:
This test isn't checking the most important file, `index.js`, which will not have it, which is why this PR will not work